src/Symfony/Component/Validator/Constraints/Timezone.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ * @author Hugo Hamon <hugohamon@neuf.fr>
+ */
+class Timezone extends Constraint
+{
+    public const NO_SUCH_TIMEZONE_IDENTIFIER_ERROR = '5ce113e6-5e64-4ea2-90fe-d2233956db13';
```

Comment:
Can we make the `const`s shorter? Like removing the `NO_SUCH_` prefix?

Reply:
Prefixes have been removed.